### DISCUSSION TITLE
a video showcase showing the speed of 8vim

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [8VIM/8VIM#541] [Ideas]
I tested 8vim a little but I didn't think it would be fast enough to type compared to [thumb-key](https://github.com/dessalines/thumb-key) it would be cool to have a video from someone who is already used to it with the keyboard and showing its efficiency

**Comment by MaethorNaur** [comment]
I think someone will make a video since the current one is down (@ManDay 👀 ???)
It's the first time I heard about thumb-key, but after testing it the main difference is you don't have to lift your finger off the screen to write a word. 8Vim was heavily inspired by 8pen (https://www.youtube.com/watch?v=q3OuCR0EpGo for the usage).
Personally, I have used that kind of keyboard since Windows CE. The learning curve can be steep but once you know your layout it's really fast to type in. There is of course still room for improvement.
You can even use it without looking at your screen once you master it.

No, I'm trying to sell 8Vim over thumb-key 😅 .
More seriously it could be nice to have a writing type test comparison.

**Comment by ManDay** [comment]
[8vim.webm](https://github.com/user-attachments/assets/a0f440f1-a1e4-45fe-a3a3-0a4cc9bd8b5b)

**Comment by flide** [comment]
oh this is lovely!!
can we use this video in the readme?

**Comment by ManDay** [comment]
Hi @flide, yes of course, thank you! It's yours now. I attach the [the "source"](https://github.com/user-attachments/files/16250412/8vim.tar.gz) (bash script which runs ffmpeg), in case you want to change anything or re-dub it.
